tests/data.ts

```diff
@@ -0,0 +1,3 @@
+export const userName = 'Admin';
+export const userPwd = 'admin123';
```

Comment:
If those are global const you should use snake case like const USER_NAME = '...'

Reply:
fixed